internal/server/v1/firehose/mappings.go

```diff
@@ -163,7 +162,7 @@ func slugify(s string) string {
 	return s
 }
 
-func mergeMaps(m1, m2 map[string]string) map[string]string {
+func cloneMerge(m1, m2 map[string]string) map[string]string {
```

Comment:
Should we name this `cloneAndMergeMaps` ?

Reply:
done